api-lambda-ses/template.yml

```diff
@@ -0,0 +1,87 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+
+Resources:
+  RestApi:
+    Type: AWS::Serverless::Api
+    Properties:
+      StageName: s1
```

Comment:
StageName doesn't match the stage name in the deploy instructions.

Reply:
Stage name is defined as s1 everywhere